### PR TITLE
Website Forum Doc newapi apa

### PR DESCRIPTION
**Pull request by apa-odoo**
<b>Task:</b> https://www.odoo.com/web#id=12123&view_type=form&model=project.task&menu_id=3942&action=327

<b>Pad:</b> https://pad.odoo.com/p/r.xbcbss7AVs15pgHe

### CODE REVIEW

File: addons/website_forum_doc/controllers/main.py [hunk 1]
```diff
@@ -1,22 +1,17 @@
...
-        cr, uid, context, toc_id = request.cr, request.uid, request.context, False
         if toc:
             sections = toc.child_ids
             forum = toc.forum_id
         else:
-            toc_obj = request.registry['forum.documentation.toc']
-            obj_ids = toc_obj.search(cr, uid, [('parent_id', '=', False)], context=context)
-            sections = toc_obj.browse(cr, uid, obj_ids, context=context)
+            sections = request.env['forum.documentation.toc'].search([('parent_id', '=', False)])
             forum = sections and sections[0].forum_id or False
```
Comment (atp-odoo) [inline addons/website_forum_doc/controllers/main.py]: search with limit=1 so no need of sections[0]

File: addons/website_forum_doc/models/documentation.py [hunk 2]
```diff
@@ -11,75 +12,55 @@ class Documentation(osv.Model):
...
-        reads = self.read(cr, uid, ids, ['name', 'parent_id'], context=context)
+    @api.multi
+    def name_get(self):
         res = []
-        for record in reads:
-            name = record['name']
-            if record['parent_id']:
-                name = record['parent_id'][1]+' / '+name
-            res.append((record['id'], name))
+        for record in self.search([]):
```
Comment (atp-odoo) [inline addons/website_forum_doc/models/documentation.py]: why self.search ? for record in self

Comment (apa-odoo) [inline addons/website_forum_doc/models/documentation.py]: ok,you are right.

File: addons/website_forum_doc/controllers/main.py (hunk 1 above)
Comment (apa-odoo) [inline addons/website_forum_doc/controllers/main.py]: Search with limit=1return only one section, so it show only one Documentation. 
In value there is pass multi sections.

**Comment by hmo-odoo** [comment]
continue on PR: https://github.com/odoo/odoo/pull/8481/files

**Pull request closed without merging**